pyriemann_qiskit/datasets/utils.py

```diff
@@ -98,12 +98,19 @@ def get_mne_sample(n_trials=10, include_auditory=False):
     return X, y
 
 
-def get_qiskit_dataset():
+def get_qiskit_dataset(n_samples=30):
```

Comment:
Wouldn't be confusing for someone who wants the entire dataset to get only 30 samples when calling the function without parameters?

Reply:
This dataset is generated. So you have to specify somehow the number of samples you want.

Reply:
I see. Then the name could be "generate_qiskit_dataset".

Reply:
Yes, good point.